modules/mod_translation/mod_translation.erl

```diff
@@ -301,11 +301,18 @@ event(#postback{message={language_delete, Args}}, Context) ->
             z_render:growl_error(?__("Sorry, you don't have permission to change the language list.", Context), Context)
     end;
 
-event(#postback{message={language_enable, Args}}, Context) ->
+event(#postback{message={language_status, Args}}, Context) ->
     case z_acl:is_allowed(use, ?MODULE, Context) of
         true ->
             {code, Code} = proplists:lookup(code, Args),
-            language_enable(Code, z_convert:to_bool(z_context:get_q("triggervalue", Context)), Context),
+            case z_convert:to_binary(z_context:get_q("triggervalue", Context)) of
```

Comment:
Remove newline?

Reply:
Which newline?

Reply:
There is an newline between `case of` and `<<enabled>`?

Reply:
There is not!